shared/test/otSdkDriver_test.js

```diff
@@ -219,12 +274,103 @@ describe("loop.OTSdkDriver", function() {
           scrollWithPage: true
         }
       };
+
       driver.startScreenShare(options);
 
       sinon.assert.calledWithExactly(driver._noteSharingState, "browser", true);
     });
   });
 
+  describe("#_onScreenSharePublishComplete", function() {
```

Comment:
Ok, I missed this one as well. Please can you merge this describe with the existing `describe("#startScreenShare")` one.